ios/FluentUI/SegmentedControl/SegmentPillButton.swift

```diff
@@ -41,6 +41,7 @@ class SegmentPillButton: UIButton {
                                                               trailing: horizontalInset)
         configuration.background.backgroundColor = .clear
         configuration.baseForegroundColor = tokenSet[.restLabelColor].uiColor
+        configuration.background.cornerRadius = 16
```

Comment:
As mentioned by @laminesm, I'd like to see this stored as a named constant somewhere--either as a token constant, or potentially added as a `CornerRadiusToken.radius160`, though that would require updating the token design library.

Reply:
Alternatively, can we just use `CornerRadiusToken.radiusCircular` for the same effect?

Reply:
`CornerRadiusToken.radiusCircular` would result in a corner radius that's too large. It's already stored in `SegmentedControl.Constants` so I'll move that to `SegmentedControlTokenSet` and access it there.